assets/js/components/ExecutionResults/ChecksResultFilters.jsx

```diff
@@ -4,14 +4,25 @@ import Filter from '@components/Table/Filter';
 
 export const RESULT_FILTER_FIELD = 'result';
 
-function ChecksResultFilters({ onChange }) {
+const getFilters = (savedFilters, searchParams) =>
+  savedFilters &&
+  savedFilters.length >= 0 &&
+  searchParams.getAll('health').length === 0
```

Comment:
We could extract this `searchParams.getAll('health')` outside of `getFilters` function. 
I am aware it wouldn't change much :sweat_smile:, so not a biggie. However if you want to give it a go here's the changes it would require.

```js
function ChecksResultFilters() {
  //...
  const qsFilters = searchParams.getAll('health');

  useEffect(() => {
    const selectedFilters = getFilters(savedFilters, qsFilters);
    
    setFiltersForField({
        //...
    });
  }, [searchParams, savedFilters]);

  //...
  return (
    <div className="flex">
      <Filter
        //...
        value={getFilters(savedFilters, qsFilters)}
      />
    </div>
  );
}
```

Reply:
I tried and the component exploded in my face :sweat_smile: